docs/ref/models/querysets.txt

```diff
@@ -2219,7 +2219,7 @@ prepared to handle the exception if you are using manual primary keys.
 
 *Asynchronous version*: ``aget_or_create()``
 
-A convenience method for looking up an object with the given ``kwargs`` (may be
+A convenient method for looking up an object with the given ``kwargs`` (it may be
```

Comment:
"convenience method" is a common programming term: https://en.wikipedia.org/wiki/Convenience_function.